### PR TITLE
proposed fix to issue #593

### PR DESCRIPTION
**Pull request by pgkirsch**
Ready for review @whoburg @bqpd

### CODE REVIEW

File: gpkit/constraints/signomial_program.py [hunk 1]
```diff
@@ -57,6 +57,7 @@ def __init__(self, cost, constraints, substitutions=None, verbosity=2):
     Constraints, since Models without Signomials have global solutions and can
     be solved with 'Model.solve()'.""")
         self.gps = []
+        self.constraints = constraints
```
Comment (bqpd) [inline gpkit/constraints/signomial_program.py]: This shouldn't be necessary, because...

File: gpkit/constraints/signomial_program.py [hunk 2]
```diff
@@ -126,6 +127,11 @@ def localsolve(self, solver=None, verbosity=1, x0=None, rel_tol=1e-4,
         self.sens_from_gpconstr(gp.constraints,
                                 result["sensitivities"]["constraints"],
                                 result["sensitivities"]["constants"])
+
+        for constraint in self.constraints:
```
Comment (bqpd) [inline gpkit/constraints/signomial_program.py]: `process_result` is inherited and so you should just be able to call `self.process_result(result)`

**Comment by whoburg** [comment]
retest this please

**Comment by whoburg** [comment]
looks good to me, fine to merge

**Pull request merged**